.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
The `.idea/` directory shouldn't be part of the PR

Reply:
awkward...